CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Release built: _not published yet_
   - Dockerfiles no longer specify custom `app` user as it comes built-in with official base images.
   - Removed now-obsolete or no-longer-needed code.
   - Prometheus integration exposes new built-in metric `httpclient_request_duration_seconds_bucket` for all registered HTTP client. 
+- Introduced new configuration parameter `LedgerConfirmation.MinCommitBatchSize`. It controls the minimum batch size of transactions that will be processed, set by default to 1. 
```

Comment:
I'd probably suggest not to announce that as it turned out it's going ot be rather "internal feature".

Reply:
removed